### PR TITLE
feat: add page descriptions

### PR DESCRIPTION
**Pull request by itai-rozen**
# Description
1. titles in navbar are now the same as page titles
2. added descriptions below page titles in hebrew & english

## screenshots
image
image
image
image

<paste here>

### CODE REVIEW

**Comment by NoamGaash** [comment]
thanks!
please address the building  & linting issue (you may run `npm run lint` locally or visit [this link](https://github.com/hasadna/open-bus-map-search/actions/runs/7775652806/job/21201860523?pr=452))

all :x: should be solved
image

**Comment by itai-rozen** [comment]
not sure what went wrong here.. any help?

**Comment by shootermv** [comment]
Will be glad to try help you. Do you have a discord ?

**Comment by itai-rozen** [comment]
@johnglaire

‫בתאריך יום ב׳, 12 בפבר׳ 2024 ב-6:57 מאת ‪Moshe‬‏ ***@***.***
‬‏>:‬

**Comment by NoamGaash** [comment]
it said that the operation was canceled. I'll try to re-run

File: tests/menu.spec.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ test('menu', async ({ page }) => {
   await page.goto('/')
   await expect(page.locator('h1')).toContainText('דאטאבוס')
   const menuItemsInOrder = [
-    'קיום נסיעות',
-    'לוח זמנים היסטורי',
-    'נסיעות שלא יצאו',
-    'דפוסי נסיעות שלא יצאו',
-    'מפה בזמן אמת',
+    'ביצועי תחבורה ציבורית',
```
Comment (NoamGaash) [inline tests/menu.spec.ts:7]: אנחנו לא מודדים ביצועים - האם הנסיעה איחרה, התעכבה וכו'. רק את עצם הקיום או אי הקיום שלה

**Comment by NoamGaash** [comment]
would you please solve the conflicts and add some screenshots?
thanks

**Review by NoamGaash — APPROVED** [review]
Thanks!

**Comment by aviv1620** [comment]
Screenshot 2024-03-11 165741

**Pull request merged**